### PR TITLE
Added a list of extensions to pre-install in Codespaces

### PR DESCRIPTION
**Pull request by okineadev**
I added all extensions from `.vscode/extensions.json` to `devcontainer.json` so that they are preinstalled in **Codespace**

### CODE REVIEW

File: .devcontainer/ubuntu18.04/devcontainer.json [hunk 1]
```diff
@@ -1,9 +1,38 @@
...
   "workspaceMount": "source=${localWorkspaceFolder},target=/home/linuxbrew/.linuxbrew/Homebrew,type=bind,consistency=cached",
+
   "onCreateCommand": ".devcontainer/on-create-command.sh",
+
+  "customizations": {
+    "vscode": {
+      "settings": {
+        // Turns off the display of CPU and battery frequency (because it's useless)
+        // This setting is only required for GitHub Codespaces
+        "resmon.show.battery": false,
```
Comment (fxcoudert) [inline .devcontainer/ubuntu18.04/devcontainer.json:16]: I don't think it hurts, really. Let's not disable those and keep the configuration as simple as possible.

File: .devcontainer/ubuntu18.04/devcontainer.json [hunk 2]
```diff
@@ -1,9 +1,38 @@
+// For format details, see https://aka.ms/devcontainer.json.
```
Comment (fxcoudert) [inline .devcontainer/ubuntu18.04/devcontainer.json:1]: I don't feel we should duplicate information into two files.

Comment (okineadev) [reply .devcontainer/ubuntu18.04/devcontainer.json:1]: Needed to quickly find documentation on `devcontainer.json`.

File: .devcontainer/ubuntu18.04/devcontainer.json (hunk 1 above)
Comment (okineadev) [reply .devcontainer/ubuntu18.04/devcontainer.json]: Done - [`39aaf33` (#17298)](https://github.com/Homebrew/brew/pull/17298/commits/39aaf333403e8b4d636ff6fc08650e9f67f9ff77)

**Review by MikeMcQuaid — APPROVED** [review]
Looks good, thanks @okineadev!

**Pull request merged**